src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/jpa/entity/Cas2ApplicationEntity.kt

```diff
@@ -49,8 +49,14 @@ SELECT
     a.crn,
     CAST(a.created_by_user_id AS TEXT) as createdByUserId,
     a.created_at as createdAt,
-    a.submitted_at as submittedAt
+    a.submitted_at as submittedAt,
+    asu.label as latestStatusUpdate
```

Comment:
Ah if that ID is a dynamically generated uuid I don't think that'll work as each environment will have a different ID for the same status. 

It could work if we did have something similar to `name` persisted in the database:

```
        name = "moreInfoRequested",
        label = "More information requested",
```

Then we needed to tweak the human naming to for example "Extra information needed", we'd change the `label` but keep the 'name' the same.

What do you think about removing the surfacing of the ID here (assuming I've got that bit right) and then holding this work whilst we first make the change to make persist the name? If we merge this first then the UI will plug into the label and it will be harder to justify.

Reply:
ah the ID is hardcoded in `Cas2ApplicationStatusSeeding` - it's always the same ID regardless of environment. But it is less human-readable than the `name` so would be less easy to parse as a frontend dev.

Reply:
Ah so it's not a dynamic id 👍 

[From looking at that seeding class it looks like `name` is persisted](https://github.com/ministryofjustice/hmpps-approved-premises-api/blob/899aab2cc3f39e9aca740366b27c38c19f202dc3/src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/model/reference/Cas2ApplicationStatusSeeding.kt#L12)(?), are we able to return that after all?

I've let Pat know the intention to hook into the ID instead of the label for now.

Reply:
The `name` is persisted on the list of possible Statuses but sadly it looks like we are only saving the `status_id` against a Status Update, not the name :( 
(It's a bit confusing.. A Status Update is basically a record of adding a Status to an Application. )
We would have to do some migration work to add the `status_name` to the `cas_2_status_updates` table, so that all historical status_updates had the field, and then update the types and transformer... I'm not sure whether it's worth it?